pom.xml

```diff
@@ -508,6 +508,7 @@
     <!-- keep in sync with PlatformDependent#ALLOWED_LINUX_OS_CLASSIFIERS -->
     <os.detection.classifierWithLikes>fedora,suse,arch</os.detection.classifierWithLikes>
     <tcnative.artifactId>netty-tcnative</tcnative.artifactId>
+    <!-- IMPORTANT: This needs to be kept in sync with the version used in the bom-ssl/pom.xml file -->
     <tcnative.version>2.0.41.Final</tcnative.version>
```

Comment:
Can netty use `netty-bom-ssl` instead of defining another constant for that?

Reply:
I think this not works .. what exactly do you have in mind ?

Reply:
Like depending on the `bom-ssl` and not specifying a version for `tcnative`, letting it to be inferred through the bom

Reply:
From what I know this shouldn't be done ... I also don't think it's a big problem to just keep these in sync

Reply:
If the `bom-ssl` is not used by this repo, it can be moved to https://github.com/netty/netty-tcnative